### PR TITLE
Improve development documentation: DB container initialization

### PR DESCRIPTION
**Pull request by midmarch**
1. Split database (type) configuration from DB container configuration
2. Replace MSSQL specific instructions with DB independent instructions

### CODE REVIEW

File: docs/dev-setup.md [hunk 1]
```diff
@@ -110,13 +109,6 @@ To log in as one of the base data users, when prompted for a username and passwo
 1. If you want to see the app running, continue to the [React Frontend](#react-frontend) instructions.
 
 ## Testing
-### Initial Setup Test Database
-After successfully creating and building the MSSQL Docker container it is possible to create a dedicated container for testing. Use the `-PtestEnv` property to access the test environment settings in `gradle`.
-1. Create the MSSQL Docker container and test database `./gradlew -PtestEnv dockerCreateDB`
-1. Start the MSSQL Docker container: `./gradlew -PtestEnv dockerStartDB`
-1. Wait until the container is fully started (can be done automatically with `./gradlew -PtestEnv dbWait`), then run `./gradlew -PtestEnv dbMigrate`
```
Comment (midmarch) [inline docs/dev-setup.md:117]: Is the `-PtestEnv` required?

File: docs/dev-setup.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ The frontend is run with [`yarn`](https://yarnpkg.com/).  We recommend running t
 ### Prerequisites
 1. Make sure you can [manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
 
-### Initial Setup
+### Setup Test Database
```
Comment (gjvoosten) [inline docs/dev-setup.md:38]: This setup describes setting up the development database.

File: docs/dev-setup.md [hunk 3]
```diff
@@ -110,13 +109,6 @@ To log in as one of the base data users, when prompted for a username and passwo
 1. If you want to see the app running, continue to the [React Frontend](#react-frontend) instructions.
 
 ## Testing
-### Initial Setup Test Database
-After successfully creating and building the MSSQL Docker container it is possible to create a dedicated container for testing. Use the `-PtestEnv` property to access the test environment settings in `gradle`.
-1. Create the MSSQL Docker container and test database `./gradlew -PtestEnv dockerCreateDB`
-1. Start the MSSQL Docker container: `./gradlew -PtestEnv dockerStartDB`
-1. Wait until the container is fully started (can be done automatically with `./gradlew -PtestEnv dbWait`), then run `./gradlew -PtestEnv dbMigrate`
-1. Seed initial data - MSSQL: `./gradlew -PtestEnv dbLoad`.
-1. Run `./gradlew -PtestEnv build` to download all dependencies and build the project.
```
Comment (gjvoosten) [inline docs/dev-setup.md:119]: This setup describes setting up the test database (hence the `-PtestEnv`, which selects a different container etc.).

**Review by gjvoosten — CHANGES_REQUESTED** [review]

File: docs/dev-setup.md (hunk 3 above)
Comment (midmarch) [reply docs/dev-setup.md:119]: Yeah, indeed I missed the point between the distinct between test-DB and the development-DB.

**Review by gjvoosten — APPROVED** [review]

**Review by gjvoosten — APPROVED** [review]

**Pull request merged**